packages/lib-user/dev/components/App/App.js

```diff
@@ -0,0 +1,134 @@
+import oauth from 'panoptes-client/lib/oauth.js'
```

Comment:
If this is the main component intended to be imported into app-root, I think it will need to have `'use client'` at the top of the file because it has `useState` and `useEffect`. Then, we won't need to create a container component in app-root simply for indicating this is a client component.